crash_similarity.py

```diff
@@ -20,6 +21,7 @@
 from download_data import download_stack_traces_for_signature
 
 pyximport.install()
+logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
Let's not set `level` here and in the other `basicConfig` calls. The level will be the one chosen by calling the `logging` functions (`logging.info`, `logging.debug`, etc.).